packages/components/src/schemas/HomePageAPI.schema.ts

```diff
@@ -0,0 +1,12 @@
+import { Quote } from './Quote.schema';
```

Comment:
the name should be HomePage, not HomePageAPI

Reply:
done